tests/system/feature_store/test_spark_snowflake.py

```diff
@@ -208,3 +213,145 @@ def test_purge_with_missing_attribute(self):
             match=".*some attributes are missing.*",
         ):
             fake_target.purge()
+
+    def test_feature_set_wrong_letters_case(self):
```

Comment:
since the fix is generic (not specific to source) no need for specific test for snowflake.